src/Symfony/Component/Process/ExecutableFinder.php

```diff
@@ -46,7 +46,7 @@ public function addSuffix($suffix)
      * @param string $default   The default to return if no executable is found
      * @param array  $extraDirs Additional dirs to check into
      *
-     * @return string The executable path or default value
+     * @return string|null The executable path or default value
```

Comment:
If I don't miss anything, this can only happen when `$default` is returned. We may have wanted to fix the param for this argument too then. ;)

Reply:
@xabbuh 

Do you mean

```diff
diff --git a/src/Symfony/Component/Process/ExecutableFinder.php b/src/Symfony/Component/Process/ExecutableFinder.php
index ad2c76ab60..f4ce96b1aa 100644
--- a/src/Symfony/Component/Process/ExecutableFinder.php
+++ b/src/Symfony/Component/Process/ExecutableFinder.php
@@ -42,9 +42,9 @@ class ExecutableFinder
     /**
      * Finds an executable by name.
      *
-     * @param string $name      The executable name (without the extension)
-     * @param string $default   The default to return if no executable is found
-     * @param array  $extraDirs Additional dirs to check into
+     * @param string|null $name      The executable name (without the extension)
+     * @param string      $default   The default to return if no executable is found
+     * @param array       $extraDirs Additional dirs to check into
      *
      * @return string|null The executable path or default value
      */
```

?

Reply:
Yes